ambry-tools/src/main/java/com/github/ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -214,11 +217,13 @@ static void updateResourceIdealState(String destZkString, String destClusterName
    */
   static IdealState buildIdealState(String resource, Set<String> partitionSet) {
     FullAutoModeISBuilder builder = new FullAutoModeISBuilder(resource);
-    builder.setStateModel(LeaderStandbySMD.name);
+    builder.setStateModel(OnlineOfflineSMD.name);
     for (String partition : partitionSet) {
       builder.add(partition);
     }
-    builder.setRebalanceStrategy(CrushEdRebalanceStrategy.class.getName());
```

Comment:
I suggest keeping this unless we have another of piece of code setting this for new resource, which I didn't see here.

Reply:
fixed. Interestingly the default rebalance strategy (which is Waged strategy i guess), isnt able to finalize a partition assignment in some cases. This was the reason the tests were failing too.